2024-10-26 06:47:12,840 INFO    MainThread:25421 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-10-26 06:47:12,840 INFO    MainThread:25421 [wandb_setup.py:_flush():76] Configure stats pid to 25421
2024-10-26 06:47:12,840 INFO    MainThread:25421 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-10-26 06:47:12,840 INFO    MainThread:25421 [wandb_setup.py:_flush():76] Loading settings from /root/workspace/hyperlink-Traceability/wandb/settings
2024-10-26 06:47:12,840 INFO    MainThread:25421 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-10-26 06:47:12,840 INFO    MainThread:25421 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-10-26 06:47:12,840 INFO    MainThread:25421 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/root/workspace/hyperlink-Traceability/main.py', 'program': '/root/workspace/hyperlink-Traceability/main.py'}
2024-10-26 06:47:12,841 INFO    MainThread:25421 [wandb_init.py:_log_setup():524] Logging user logs to /root/workspace/hyperlink-Traceability/wandb/run-20241026_064712-tj23rghc/logs/debug.log
2024-10-26 06:47:12,841 INFO    MainThread:25421 [wandb_init.py:_log_setup():525] Logging internal logs to /root/workspace/hyperlink-Traceability/wandb/run-20241026_064712-tj23rghc/logs/debug-internal.log
2024-10-26 06:47:12,841 INFO    MainThread:25421 [wandb_init.py:init():564] calling init triggers
2024-10-26 06:47:12,841 INFO    MainThread:25421 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-10-26 06:47:12,841 INFO    MainThread:25421 [wandb_init.py:init():614] starting backend
2024-10-26 06:47:12,841 INFO    MainThread:25421 [wandb_init.py:init():618] setting up manager
2024-10-26 06:47:12,843 INFO    MainThread:25421 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-26 06:47:12,844 INFO    MainThread:25421 [wandb_init.py:init():624] backend started and connected
2024-10-26 06:47:12,851 INFO    MainThread:25421 [wandb_init.py:init():716] updated telemetry
2024-10-26 06:47:12,852 INFO    MainThread:25421 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-10-26 06:47:33,659 WARNING MainThread:25421 [wandb_init.py:init():1192] interrupted
Traceback (most recent call last):
  File "/root/miniconda3/envs/mmseg/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1170, in init
    run = wi.init()
  File "/root/miniconda3/envs/mmseg/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 752, in init
    result = run_init_handle.wait(
  File "/root/miniconda3/envs/mmseg/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/root/miniconda3/envs/mmseg/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/root/miniconda3/envs/mmseg/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/root/miniconda3/envs/mmseg/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/root/miniconda3/envs/mmseg/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
